### PR TITLE
v1.23 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
 
+## 1.23.0 Release
+- Release Date: September 16, 2019
+- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Bug Fixes
```
Comment (esethna) [inline CHANGELOG.md]: @amyblais can we please order these roughly by impact?

Comment (amyblais) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+ - Fixed an issue where some Giphy actions were not working in ephemeral posts on mobile.
+ - Fixed an issue where the download step was missing when previewing a video right after posting it.
```
Comment (enahum) [inline CHANGELOG.md]: There is no download step when the video was posted from the mobile app. It was intentionally removed as you don’t need to download something that you just uploaded

**Comment by amyblais** [comment]
These are all the fixes merged for v1.23 so far. I add to the list when PRs get merged.

**Comment by amyblais** [comment]
@esethna Can you help review this?

**Comment by amyblais** [comment]
@enahum Can you help merge this?

**Pull request merged**